src/components/DocumentationTopic/ViewMore.vue

```diff
@@ -0,0 +1,43 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2023 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <div class="view-more-link">
+    <router-link
+      :to="url"
+      class="base-link"
+    >
+      View more
```

Comment:
I would make this a slot or prop, so its easy to override if needed.

Reply:
I used a slot before, but Marcus suggested to use a themed import for the `ViewMore` component. I think this method does make more sense, since we avoid passing the text as a prop to `DocumentationTopic`. What do you think?

Reply:
Both would probably be the ideal—that way another app could theme this button and easily change _only_ the text.